usaspending_api/api_contracts/contracts/v2/agency/toptier_code/obligations_by_type.md

```diff
@@ -0,0 +1,58 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Obligations [/api/v2/agency/{toptier_code}/obligations_by_type/{?fiscal_year}]
+
+This endpoint is used to power USAspending.gov's agency profile pages.
+
+## GET
+
+This endpoint returns a breakdown of obligations by award type (contracts, IDVs, grants, loans, direct payments, other) within a fiscal year.
+
++ Parameters
+    + `toptier_code`: 086 (required, number)
+        The toptier code of an agency (could be a CGAC or FREC) so only numeric character strings of length 3-4 are accepted.
+    + `fiscal_year`: 2017 (required, number)
```

Comment:
Did we determine the fiscal years for all of these endpoints will be required? i.e. No default value?

Reply:
Who was making that determination? @tony-sappe

Reply:
There was a discussion on API design during sprint planning. I don't recall what everyone's perspectives were

Reply:
FWIW, the v2 agency profile will default to filtering on the latest FY based on `v2/references/submission_periods/`

Reply:
Discussed in slack, endpoint will allow `fiscal_year` to be optional and default to the current fiscal year for the calculation